lib/slurp/logs/log_fetcher.ex

```diff
@@ -81,8 +85,14 @@ defmodule Slurp.Logs.LogFetcher do
             with {:ok, event} <-
                    Slurp.Adapter.deserialize_log_event(state.blockchain, log, subscription) do
               {mod, func, extra_args} = subscription.handler
-              args = [state.blockchain, log, event] ++ extra_args
-              apply(mod, func, args)
+              args = [state.blockchain, log, event]
+              if subscription.address != "" do
+                args = args ++ [subscription.address] ++ extra_args
```

Comment:
I know this looks terrible, Im open to suggestion on how you would write this better

Reply:
`address` is available on the `log` parameter https://eth.wiki/json-rpc/API#returns-43. We don't need to pass it along

Reply:
I dont know why I didnt see this. thanks